apps/docs/src/app/[...slug]/layout.tsx

```diff
@@ -15,6 +15,30 @@ type DocumentLayoutProps = {
   };
 };
 
+export async function generateMetadata({
+  params,
+}: {
+  params: { slug: string[] };
+}) {
+  const document = documents.getBySlug({ slug: params.slug });
```

Comment:
could add a `documents.getMeta({ slug })` and `documents.getTitle({ title })` or similar to centralise the suffix

Reply:
Good idea @ianlnf . I'll create the helpers. But I'm wondering if it wouldn't make more sense to add the suffix directly in the frontmatter since that's the place where we have the metadata content ?

Reply:
yes that's another idea for MDX files, if a calculated title field can be used, but the title helper would still be required to use in custom pages and the calculated field